src/timedata.cpp

```diff
@@ -98,11 +99,12 @@ void AddTimeData(const CNetAddr& ip, int64_t nOffsetSample)
         }
 
         if (LogAcceptCategory(BCLog::NET)) {
+            std::string logMessage{"time data samples: "};
             for (const int64_t n : vSorted) {
-                LogPrint(BCLog::NET, "%+d  ", n); /* Continued */
+                logMessage += strprintf("%+d  ", n);
             }
-            LogPrint(BCLog::NET, "|  "); /* Continued */
-            LogPrint(BCLog::NET, "nTimeOffset = %+d  (%+d minutes)\n", nTimeOffset, nTimeOffset / 60);
+            logMessage += strprintf("|  median offset = %+d  (%+d minutes)", nTimeOffset, nTimeOffset / 60);
+            LogPrint(BCLog::NET, "%s\n", logMessage);
```

Comment:
Inside the `if (LogAcceptCategory(BCLog::NET))` conditional, the check is already done so can just use `LogPrintf`.

Reply:
I think it's still better to use `LogPrint(BCLog::NET, ...)`, since that makes it easier to grep for lines that log to a specific category. If we have more structured logging in future, then it'd be better if logs were logging to the correct category.

Reply:
that makes sense to me - e.g. at some point we might decide to include the log category in the log entry in some form - so I'll keep it.

Reply:
> I think it's still better to use `LogPrint(BCLog::NET, ...)`, since that makes it easier to grep for lines that log to a specific category.

Grep-ability (grepness?) is essential but a comment on the same line can do that too.  In any case, it doesn't need to be done in this pach.